infrastructure/ai/llm_client.py

```
@@ -54,21 +60,30 @@ def _get_base_url(self) -> Optional[str]:
+        # 支持 Prompt 对象和字符串两种输入
+        if isinstance(prompt, Prompt):
+            prompt_obj = prompt
+        else:
+            # 通过模板加载器获取默认系统提示
+            try:
+                from infrastructure.ai.prompt_template_loader import PromptTemplateLoader
+                loader = PromptTemplateLoader.get_instance()
+                default_system = loader.render_with_fallback(
+                    "llm_default", "system", _DEFAULT_SYSTEM_FALLBACK,
+                )
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)